R/baggr_compare.R

```diff
@@ -57,33 +87,41 @@
 
 baggr_compare <- function(...,
                           what    = "pooling",
-                          compare = "groups",
-                          style   = "areas",
-                          arrange = "single") {
+                          compare = "groups") {
```

Comment:
I think here the idea was to also have the same option as in `baggr_plot`, `hyper = TRUE`. It would plot the mean trt effect under all group-specific estimates (if `compare = "groups"`). 

Would it also be possible to add `transform` argument that works the same way as in `baggr_plot`?

If easy to add, please do, otherwise please just log as an issue and I'll make a separate PR.